### ISSUE TITLE
508-defect-3 [COGNITION, SCREENREADER]: Lists SHOULD be announced as lists

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#18415]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I want to know how many list item exists and where I am within that list to better help me navigate through the experience.

## Details

Due to the removed list styling, Safari/VO will not recognize and announce the frequently used forms and more information lists as lists. This also makes grouped items wrapped within `<li>` s to not be announced as being related/grouped.

## Acceptance Criteria

- [ ] Safari/VO announces lists as a list along with list items

## Environment

* Operating System: Mac
* Browser: Safari
* Screenreading device: VoiceOver
* Server destination: staging

## Steps to Recreate

1. Enter `<https://staging.va.gov/find-forms/>` in browser
2. Start screenreading device listed in Environment
3. Navigate to the frequently used forms list
4. Verify that VoiceOver does not announce it as a list

## Proposed Solution 

As covered by Scott O'Hara, the fix should be to add an aria `role="list"` to the `<ul>`

## WCAG or Vendor Guidance

* ["Fixing" Lists by Scott O'Hara](https://www.scottohara.me/blog/2019/01/12/lists-and-safari.html)

## Screenshots or Trace Logs

<img width="1340" alt="Screen Shot 2021-01-12 at 11 19 22 AM" src="https://user-images.githubusercontent.com/14154792/104343220-ac408000-54c9-11eb-909d-2e5e43a715e5.png">

**Comment by MarciMcGuireGCIO** [comment]
@joshkimux , @1Copenut  -- Reading through the accessibility issues, I think that most of them were recorded as part of the MVP.  If I determine they are duplicates, can I close?  Thx!

**Comment by MarciMcGuireGCIO** [comment]
Moving all outstanding Find a VA Form 508 issues into a single Epic to clean up backlog.

**Comment by zacharymorel** [comment]
@joshkimux I added the role="list" attributes for <ul> and role="listitem" from the nested <li>'s. However, the browsers still read the text **File a VA disability claim** as a text element when navigating. 
![Screen Shot 2021-03-23 at 11 07 21 AM](https://user-images.githubusercontent.com/26075258/112169074-3e4ebe00-8bc8-11eb-8ae6-8c2740fbec60.png)


A win though is now screen reader on safari announces the <ul> as a list with 3 elements.  Is this okay enough? 
![Screen Shot 2021-03-23 at 11 07 00 AM](https://user-images.githubusercontent.com/26075258/112169221-63dbc780-8bc8-11eb-9a4c-3e602141d93b.png)

**Comment by joshkimux** [comment]
@zacharymorel No need to add in `role="listitem"` to the children! Just adding `role="list"` to the `ul` will do the trick and we'll be able to close this out 😄

**Comment by zacharymorel** [comment]
@joshkimux this fix is live on staging!

**Comment by joshkimux** [comment]
Confirmed this has been fixed. Thank you!
<img width="1108" alt="Screen Shot 2021-03-24 at 4 52 37 PM" src="https://user-images.githubusercontent.com/14154792/112381787-5fdfa080-8cc1-11eb-96b2-85f4f7b7e732.png">
